### PR TITLE
Feature/interoperability outside germany screen (EXPOSUREAPP-2264)

### PR DESCRIPTION
**Pull request by BMItr**
[Android, Interop] Screen: Target > Germany (traveling outside of Germany?)

## Relates to
https://jira.itc.sap.com/browse/EXPOSUREAPP-1975  (Overall)
https://jira.itc.sap.com/browse/EXPOSUREAPP-2264 (this Subtask)

## Description
Contains Fragment, Layouts, Viewmodel for corresponding task (Traveling outside of Germany -> Ja, Nein, Keine Angabe)

## Quicktest
Quicktest is available in
Commit: e800f98be88f704e605db804bea3a189392b59de [e800f98]
Just checkout this commit, build it, click on the menu button and select the interop fragment you would like to see

## Note
Keep in mind, that all interop subtasks / screens are devloped independently. The "Marriage" is done seperatly by followup task.

### CODE REVIEW

**Comment by MertSafter** [comment]
LGTM

**Review by mertsafter — COMMENTED** [review]
LGTM

**Review by mertsafter — DISMISSED** [review]
👍

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionOutOfGermanyViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    // @JvmField could be removed when using @get:Rule
+    @Rule
+    @JvmField
+    var instantTaskExecutorRule: InstantTaskExecutorRule = InstantTaskExecutorRule()
+
+    @Test
+    fun textButtonClickedNone() {
+        submissionOutOfGermanyViewModel.buttonClicked(ButtonClicked.NONE)
+
+        submissionOutOfGermanyViewModel.yesAnswerActive.value?.let { assertFalse(it) }
```
Comment (d4rken) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionOutOfGermanyViewModelTest.kt:23]: `?` to `!!` so tests fail on null.

Same for `?` below :point_down:

**Review by d4rken — CHANGES_REQUESTED** [review]

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionOutOfGermanyViewModelTest.kt (hunk 1 above)
Comment (BMItter) [reply Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionOutOfGermanyViewModelTest.kt]: I wanted to change it to !! but I discovered a bigger issue behind regarding transformations which took me some brain. I rewrote it and its now safe and will fail on null.

**Comment by BMItter** [comment]
Due Storychange this PR is obsolet. Very low probability to come back ever.

**Pull request closed without merging**